tests/test_stream_daq.py

```diff
@@ -1,19 +1,33 @@
 import pdb
 
 import pytest
+import pandas as pd
 
 from miniscope_io.stream_daq import StreamDevConfig, StreamDaq
 from miniscope_io.utils import hash_video, hash_file
 from .conftest import DATA_DIR, CONFIG_DIR
 
 
+@pytest.fixture()
+def default_streamdaq(set_okdev_input) -> StreamDaq:
+
+    test_config_path = CONFIG_DIR / "stream_daq_test_200px.yml"
+    daqConfig = StreamDevConfig.from_yaml(test_config_path)
+
+    data_file = DATA_DIR / "stream_daq_test_fpga_raw_input_200px.bin"
+    set_okdev_input(data_file)
+
+    daq_inst = StreamDaq(device_config=daqConfig)
+    return daq_inst
+
+
 @pytest.mark.parametrize(
     "config,data,video_hash_list,show_video",
     [
         (
             "stream_daq_test_200px.yml",
             "stream_daq_test_fpga_raw_input_200px.bin",
-            "[\"f878f9c55de28a9ae6128631c09953214044f5b86504d6e5b0906084c64c644c\",\"8a6f6dc69275ec3fbcd69d1e1f467df8503306fa0778e4b9c1d41668a7af4856\",\"3676bc4c6900bc9ec18b8387abdbed35978ebc48408de7b1692959037bc6274d\"] ",
+            '["f878f9c55de28a9ae6128631c09953214044f5b86504d6e5b0906084c64c644c","8a6f6dc69275ec3fbcd69d1e1f467df8503306fa0778e4b9c1d41668a7af4856","3676bc4c6900bc9ec18b8387abdbed35978ebc48408de7b1692959037bc6274d"] ',
```

Comment:
not sure when this got switched to just being a string that looks like a list of strings instead of being an actual list of strings, but this makes the test considerably weaker.

in particular, `str(x) in str(y)` checks if that substring is anywhere in the other string, but `str(x) in list(y)` checks whether exactly `str(x)` is found in the list of strings. So eg.

```python
>>> "" in '["some other string", "that shouldn\'t include", "the empty string literal"]'
True
>>> "" in ["some other string", "that shouldn\'t include", "the empty string literal"]
False
```